lib/view/item_widget.dart

```diff
@@ -40,8 +44,13 @@ class _ItemWidgetState extends State<ItemWidget> {
     return ListTile(
       leading: checkbox,
       onLongPress: _showOptionsModalSheet,
-      title: Text(widget.item.value,
-          style: itemTextStyle.copyWith(decoration: textDecoration)),
+      title: Padding(
+        padding: const EdgeInsets.only(bottom: 4),
+        child: Text(widget.item.value,
+            style: itemTextStyle.copyWith(decoration: textDecoration)),
+      ),
+      subtitle: widget.item.isScheduled ? _buildScheduledTimeStampChip() : null,
```

Comment:
This doesn't look great because the checkbox is no longer aligned with the item text. I think it is good to put the timestamp chip underneath the item text, but we might need to use our own custom combination of `InkWell`, `Column`, and `Row` to build the layout instead of using `ListTile`.